Gems/EMotionFX/Code/Tests/ActorBuilderTests.cpp

```diff
@@ -196,4 +197,47 @@ namespace EMotionFX
         EXPECT_EQ(mesh1->GetParentNode(), joint1);
         EXPECT_EQ(joint2->GetParentNode(), mesh1);
     }
+
+    class ActorBuilderPipelineTransformTestFixture : public ActorBuilderPipelineFixture
+        , public ::testing::WithParamInterface<AZ::Matrix3x4>
+    {
+    };
+
+    static const AZ::Matrix3x4 Matrix3x4s[] = {
+        AZ::Matrix3x4::CreateIdentity(),
+        AZ::Matrix3x4::CreateRotationX(-0.6f),
+        AZ::Matrix3x4::CreateFromQuaternion(AZ::Quaternion(0.24f, -0.08f, -0.48f, 0.84f)),
+        AZ::Matrix3x4::CreateTranslation(AZ::Vector3(7.9f, 2.4f, -4.6f)),
+        AZ::Matrix3x4::CreateFromQuaternionAndTranslation(AZ::Quaternion(0.12f, -0.24f, -0.72f, 0.64f), AZ::Vector3(2.3f, -5.2f, 0.7f))
+    };
+
+    TEST_P(ActorBuilderPipelineTransformTestFixture, ActorBuilder_Transforms)
+    {
+        // Set up a scene graph like this for testing
+        // root_joint
+        //     |____TransformData
+        using SceneGraph = AZ::SceneAPI::Containers::SceneGraph;
+        using namespace AZ::SceneData::GraphData;
```

Comment:
nit: Ideally best to avoid `using namespace` (as is can cause really weird subtle issues) and instead just use a using declaration `using AZ::SceneData::GraphData::TransformData` like you've done below, I'd just remove this line if possible

Reply:
changed